cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1734,6 +1793,101 @@ action.summary_index._name = cyences
 action.summary_index.category = O365
 action.summary_index.alert_name = O365 - Daily Login Failure
 
+[O365 - Exchange Rule]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule =	29 * * * *
+description = This alert will show the alerts for Exchange Rule. \
+\
+Data Collection - Office 365 management activity data (Splunk Add-on for Office 365). 
+dispatch.earliest_time = -85m@s
+dispatch.latest_time = -25m@s
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_o365` sourcetype="o365:management:activity" Workload=SecurityComplianceCenter Operation=AlertTriggered \
+| dedup AlertId \
+| spath input=Data \
+| table _time Name Severity AlertType Category f3u lon wl ad \
+| `cs_o365_exchange_rule_filter`
+action.summary_index = 1
+action.summary_index._name = cyences
+action.summary_index.category = O365
+action.summary_index.alert_name = O365 - Exchange Rule
+
+[O365 - Hourly Increase In Emails Over Baseline]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule =	29 * * * *
+description = This alert will show the hourly increase in emails over baseline. \
+\
+Data Collection - Office 365 reporting messagetrace data (Splunk Add-on for Office 365). 
+dispatch.earliest_time = -85m@s
+dispatch.latest_time = -25m@s
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_o365` sourcetype="ms:o365:reporting:messagetrace" \
+| stats count, values(Subject) as Subject by SenderAddress \
+| join type=inner SenderAddress \
+    [ search `cs_o365` sourcetype="ms:o365:reporting:messagetrace" earliest=-7d \
+    | bin span=1h _time \
+    | stats count as 7daycount by SenderAddress, _time \
+    | stats avg(7daycount) as average_emails_per_7days by SenderAddress \
+    | eval average_emails_per_7days=round(average_emails_per_7days,2) ] \
+| eval PercentIncrease = (count - average_emails_per_7days) / count * 100 \
+| where count > average_emails_per_7days AND PercentIncrease > 93 AND count > 10 \
```

Comment:
Lets create a macro for 93.  `cs_email_increase_over_baseline_limit_in_percentage`
So user can change when needed.